ui/common/css/vendor/chessground/_coords-colors.scss

```diff
@@ -1,47 +1,21 @@
 @mixin coords-colors {
-  .blue {
-    coord { text-shadow: none; }
-    .orientation-white .files coords:nth-child(2n+1),
-    .orientation-white .ranks coords:nth-child(2n),
-    .orientation-black .files coords:nth-child(2n),
-    .orientation-black .ranks coords:nth-child(2n+1) {
-      color: #DEE3E6;
-    }
-    .orientation-white .files coord:nth-child(2n),
-    .orientation-white .ranks coord:nth-child(2n+1),
-    .orientation-black .files coord:nth-child(2n+1),
-    .orientation-black .ranks coord:nth-child(2n) {
-      color: #788a94;
-    }
+    :root {
+    --color-white: #fff;
+    --color-black: #fff;
+    --shadow: 0 1px 2px #000;
```

Comment:
these names should be prefixed with `cg-coord`

Reply:
I haven't resolved that yet. By prefix, do you mean changing from `--color-X` to `--cg-coord-color-X`, or:
```
cg-coord {
    --color-white: #fff;
    --color-black: #fff;
    --shadow: 0 1px 2px #000;
}
```

Reply:
I meant renaming too `--cg-coord-color-white`